models/enum.js

```diff
@@ -0,0 +1,7 @@
+const distanceFormulaForQuery = Object.freeze({
+  'formula' : `(6371 * acos( cos( radians( ${latitude} ) ) * cos( radians( latitude ) ) * cos( radians( longitude ) - radians( ${longitude} ) ) + sin( radians( ${latitude} ) ) * sin( radians( latitude ) ) ) ) AS distance`
+});
```

Comment:
1. 우선 ENUM 타입에 대한 정의 정리가 필요할 것 같습니다.
`Enumerated type`의 줄임말로써 열거형 타입이나 데이터 집합을 가리킵니다.
enum이라는 파일명에서 관리하는 모듈과는 거리가 있어보여 수정이 필요할 것 같습니다.

따라서 지금 작성하신 코드는 ENUM으로 관리하기 보다는 해당 쿼리를 반환하는 하나의 모듈로 작성해주는 것이 좋아보입니다!

아래는 예시이니 참고해주세요!

```suggestion
function distanceFormulaForQuery(latitude, longitude) {
  const FORMULA_QUERY = `(6371 * acos( cos( radians( ${latitude} ) ) * cos( radians( latitude ) ) * cos( radians( longitude ) - radians( ${longitude} ) ) + sin( radians( ${latitude} ) ) * sin( radians( latitude ) ) ) ) AS distance`
  return FORMULA_QUERY
})
```

2. 유저의 위치 기반에 따른 거리 차이도 해당 코드로 구현할 수 있는 것인가요?